squire/util.py

```diff
@@ -0,0 +1,47 @@
+"""
+Contains various utility functions.
+"""
+
+import os
+
+def get_secret_key(filePath: str, enableMessages: bool = False) -> str:
+    """
+    Get the stored secret key from the filesystem.
+
+    If this is the first time the program is run, create one.
+    """
+
+    try:
+        secretfile = filePath
+        with open(secretfile) as f:
+            secret = f.read().strip()
+    except FileNotFoundError:
+        from django.core.management.utils import get_random_secret_key
+        if enableMessages: #pragma: debug
+            print("Hello and welcome! I think that this is the first time you are" +
+                " running me, I'm generating a new Secret Key for you to use. " +
+                "Saving it to a file for next time...")
+        secret = get_random_secret_key()
+        with open(secretfile, 'w') as f:
+            f.write(secret)
+    return secret
+
+
+def createCoverageDirectory(directory: str, enableMessages: bool = False) -> None:
```

Comment:
which naming scheme will we use: `snake_case` or `mixedCase`? This method is mixedCase, while get_secret_key is snake_case. PEP-8 prefers snake_case. What do you think?